### ISSUE TITLE
Debug logs of Production update printed to all AI logs

### ISSUE THREAD

**Issue by Cjkjvfnby** [freeorion/freeorion#521]
Debug text related to single AI printed to all AI logs. 
`AI_1.log`:

```
2016-03-12 13:18:48.344096 [debug] AI : ========= Production Update for empire: 1 ========
2016-03-12 13:18:48.345097 [debug] AI : ========= Production Update for empire: 2 ========
2016-03-12 13:18:48.345097 [debug] AI : ========= Production Update for empire: 3 ========
2016-03-12 13:18:48.345097 [debug] AI : ========= Production Update for empire: 4 ========
2016-03-12 13:18:48.346097 [debug] AI : ========= Production Update for empire: 5 ========
2016-03-12 13:18:48.346097 [debug] AI : ========= Production Update for empire: 6 ========
2016-03-12 13:18:48.346097 [debug] AI : ========= Production Update for empire: 7 ========
2016-03-12 13:18:48.346097 [debug] AI : ========= Production Update for empire: 8 ========
2016-03-12 13:18:48.346097 [debug] AI : ========= Production Update for empire: 9 ========
2016-03-12 13:18:48.347097 [debug] AI : ========= Production Update for empire: 10 ========
2016-03-12 13:18:48.347097 [debug] AI : ========= Production Update for empire: 11 ========
2016-03-12 13:18:48.347097 [debug] AI : ========= Production Update for empire: 12 ========
2016-03-12 13:18:48.347097 [debug] AI : ========= Production Update for empire: 13 ========
2016-03-12 13:18:48.347097 [debug] AI : ========= Production Update for empire: 14 ========
2016-03-12 13:18:48.348097 [debug] AI : ========= Production Update for empire: 15 ========
2016-03-12 13:18:48.348097 [debug] AI : ========= Production Update for empire: 16 ========
2016-03-12 13:18:48.348097 [debug] AI : ========= Production Update for empire: 17 ========
2016-03-12 13:18:48.348097 [debug] AI : ========= Production Update for empire: 18 ========
2016-03-12 13:18:48.348097 [debug] AI : ========= Production Update for empire: 19 ========
2016-03-12 13:18:48.349097 [debug] AI : ========= Production Update for empire: 20 ========
```

If any other error occurred on update them printed to all log too.
This is example from my **development** branch with additional entries. 
`AI_1.log`:

```
2016-03-12 13:19:18.325811 [debug] AI : ========= Production Update for empire: 1 ========
2016-03-12 13:19:18.326811 [debug] AI : ========= Production Update for empire: 2 ========
2016-03-12 13:19:18.326811 [debug] AI : ProductionQueue::Update: Simulating future turns of production queue
2016-03-12 13:19:18.326811 [debug] AI : ProductionQueue::Update: Projections took 0 microseconds with 13.4 total Production Points
2016-03-12 13:19:18.326811 [debug] AI : ========= Production Update for empire: 3 ========
2016-03-12 13:19:18.326811 [debug] AI : not enough PP to be worth simulating future turns production.  marking everything as never complete
2016-03-12 13:19:18.326811 [debug] AI : ========= Production Update for empire: 4 ========
2016-03-12 13:19:18.327811 [debug] AI : not enough PP to be worth simulating future turns production.  marking everything as never complete
2016-03-12 13:19:18.327811 [debug] AI : ========= Production Update for empire: 5 ========
2016-03-12 13:19:18.327811 [debug] AI : not enough PP to be worth simulating future turns production.  marking everything as never complete
2016-03-12 13:19:18.327811 [debug] AI : ========= Production Update for empire: 6 ========
```

**Comment by Vezzra** [comment]
Has this issue been sufficiently addressed by #1460? Can we close this as resolved?

**Comment by Morlic-fo** [comment]
Seems this was fixed by #1460 indeed.
